src/reducers/ui.js

```diff
@@ -90,6 +97,24 @@ export default function ui(stateIn, action) {
     case 'EDITOR_FOCUSED_REQUESTED_LINE':
       return state.setIn(['editors', 'requestedFocusedLine'], null);
 
+    case 'EDITOR_FOCUSED':
+      return state.setIn(
+        ['editors', 'focusedEditor'],
+        action.payload.language,
+      );
+
+    case 'EDITOR_BLURRED':
+      return state.setIn(
+        ['editors', 'focusedEditor'],
+        null,
+      );
+
+    case 'EDITOR_RESIZED':
+      return state.setIn(
+        ['editors', 'focusedSelector'],
+        null,
+      );
```

Comment:
This is to remove the highlighter in the event of the resizing. Alternatively, we could just update cover with new window size on  resize

Reply:
This is more about the resize of the entire window instead of the editors but works for both

Reply:
OK—in that case I think we should make it explicit rather than piggybacking on a resize listener that seems more or less unrelated.

Question: don’t we actually want to listen for resize inside the iframe? Seems like that’s the real question, and also I think that would catch certain cases we are missing now (e.g. moving the vertical divider changes the preview frame size).

As a corollary, I think right now a resize will fully remove the highlights; better I think to just recalculate their positions?